src/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Media Keys by Sway.fm",
-  "version": "4.42",
+  "version": "4.51",
```

Comment:
in the future don't bump the version, let me collect all changes and do that at once